Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -36,6 +36,7 @@ private struct HistoryPanelUX {
 class HistoryPanel: UIViewController, HomePanel {
     weak var homePanelDelegate: HomePanelDelegate? = nil
     var profile: Profile!
+    private var currentSyncedDevicesCount: Int? = 0
```

Comment:
Nit: If we're assigning it 0 here and a number elsewhere, it never becomes nil so the optional is meaningless. I would keep this as nil, then assign it the count later on. This way we can use the non-existence of a value as a way of checking to see if we've fetched the value yet.